.rubocop_todo.yml

```diff
@@ -0,0 +1,809 @@
+    - "app/controllers/api/v3/application_controller.rb"
+    - "app/controllers/error_controller.rb"
+    - "app/serializers/api/public/v1/serializable_location.rb"
+    - "app/serializers/api/public/v1/serializable_provider.rb"
+    - "spec/controllers/error_controller_spec.rb"
+    - "spec/requests/api/v2/access_request_spec.rb"
+    - "spec/requests/api/v2/providers/accredited_body_filter_providers_spec.rb"
+    - "spec/requests/api/v3/courses/index_spec.rb"
+    - "spec/serializers/api/public/v1/serializable_location_spec.rb"
+    - "spec/serializers/api/public/v1/serializable_provider_spec.rb"
+
+# Offense count: 3
+RSpec/AnyInstance:
+  Exclude:
+    - "spec/requests/api/v3/courses/index_spec.rb"
+    - "spec/support/with_stubbed_stdout.rb"
+
+# Offense count: 4
+RSpec/BeforeAfterAll:
+  Exclude:
+    - "spec/test_setup_specs/test_data_cache_spec.rb"
+    - "spec/support/reference_data.rb"
+
+# Offense count: 8
+RSpec/EmptyExampleGroup:
+  Exclude:
+    - "spec/docs/courses_spec.rb"
+    - "spec/docs/provider_suggestions_spec.rb"
+    - "spec/docs/providers/courses/locations_spec.rb"
+    - "spec/docs/providers/courses_spec.rb"
+    - "spec/docs/providers/locations_spec.rb"
+    - "spec/docs/providers_spec.rb"
+    - "spec/docs/subject_areas_spec.rb"
+    - "spec/docs/subjects_spec.rb"
+
+# Offense count: 2
+RSpec/ExpectOutput:
+  Exclude:
+    - "spec/services/allocation_importer_service_spec.rb"
+
+# Offense count: 20
+# Configuration parameters: Include, CustomTransform, IgnoreMethods, SpecSuffixOnly.
+# Include: **/*_spec*rb*, **/spec/**/*
+RSpec/FilePath:
+  Enabled: false
+
+# Offense count: 4
+RSpec/IteratedExpectation:
+  Exclude:
+    - "spec/models/concerns/courses/edit_options/qualifications_concern_spec.rb"
+    - "spec/models/course/publish_enrichments_spec.rb"
+    - "spec/requests/api/v2/organisations_spec.rb"
+
+# Offense count: 4
+RSpec/LeakyConstantDeclaration:
+  Exclude:
+    - "spec/lib/time_format_spec.rb"
+    - "spec/validators/email_validator_spec.rb"
+    - "spec/validators/phone_validator_spec.rb"
+    - "spec/validators/reference_number_format_validator_spec.rb"
+
+# Offense count: 3
+# Cop supports --auto-correct.
+RSpec/LetBeforeExamples:
+  Exclude:
+    - "spec/requests/api/v2/providers/update_accrediting_providers_spec.rb"
+    - "spec/requests/api/v2/providers/update_spec.rb"
+
+# Offense count: 95
+RSpec/MessageChain:
+  Exclude:
+    - "spec/services/allocation_reporting_service_spec.rb"
+    - "spec/services/course_reporting_service_spec.rb"
+    - "spec/services/provider_reporting_service_spec.rb"
+    - "spec/services/publish_reporting_service_spec.rb"
+    - "spec/services/send_welcome_email_service_spec.rb"
+
+# Offense count: 213
+# Configuration parameters: EnforcedStyle.
+# SupportedStyles: have_received, receive
+RSpec/MessageSpies:
+  Enabled: false
+
+# Offense count: 1
+RSpec/MultipleDescribes:
+  Exclude:
+    - "spec/requests/api/v3/courses/index_spec.rb"
+
+# Offense count: 1098
```

Comment:
Can you fix this 
> until I lost the will to live.

before you knock off?

Reply:
Ha ha. Next PR that one.